SubtitlesParser/Classes/Utils/SsaWrapStyle.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace SubtitlesParser.Classes.Utils
+{
+    /// <summary>
+    /// Represents a Wrap Style used by Advanced SSA
+    /// </summary>
+    // Note: the spec doc doesn't actually specify a name, just a number and a description, so I took some creative liberties
+    public enum SsaWrapStyle
+    {
+        /// <summary>
+        /// Smart wrapping, lines are evenly broken
+        /// </summary>
+        Smart = 0,
+        /// <summary>
+        /// End-of-line word wrapping, only \N breaks
+        /// </summary>
+        EndOfLine = 1,
+        /// <summary>
+        /// No word wrapping, \n \N both breaks
+        /// </summary>
+        None = 2,
+        /// <summary>
+        /// Same as Smart, but the lower line gets wider
+        /// </summary>
+        SmartWideLowerLine = 3
+    }
+
+    /// <summary>
+    ///  Extension methods for parsing to a wrap style
+    /// </summary>
+    public static class SsaWrapStyleExtensions
+    {
+        /// <summary>
+        /// Parse a string into a wrap style
+        ///
+        /// Invalid input strings will return `SsaWrapStyle.None`
+        /// </summary>
+        /// <param name="rawString">A string representation of a wrap style value</param>
+        /// <returns>A SsaWrapStyle corresponding to the value parsed from the input string</returns>
+        public static SsaWrapStyle FromString(this string rawString) =>
+            int.TryParse(rawString, out int rawInt) == false ?
+                SsaWrapStyle.None: // basically an arbitrary choice, could also throw an exception here instead
```

Comment:
Just wanted to draw your attention to this, as I'm not sure what you'd prefer the library do when it fails to parse the value. If you'd rather have it throw an exception, I can change it.

Reply:
I'm also looking at this again and realizing the `== false` is redundant since it's a ternary operator instead of the if statement I originally had... 

Will leave it unless you want it changed, or if I go in there anyways to change to throwing an exception.

Reply:
Indeed, the two options have their pros and cons.
You can leave it like this for the moment. I can't quite get cases where things could go wrong but if I have some in the future, I'll consider if the other option (throwing an exception) is more adapted.